applications/fishing-map/src/features/workspace/workspace.selectors.ts

```diff
@@ -18,20 +33,54 @@ export const selectWorkspaceViewport = createSelector([selectWorkspace], (worksp
   return workspace?.viewport
 })
 
-export const selectWorkspaceDataviews = createSelector([selectWorkspace], (workspace) => {
+export const selectWorkspaceDataviewsResolved = createSelector([selectWorkspace], (workspace) => {
   if (!workspace) return
-  // TODO resolve them here
-  return workspace.dataviews
+  // TODO move this to use-workspace helper
+  const dataviews = workspace.dataviews.flatMap((dataview) => {
```

Comment:
I'm a bit puzzled by the logic... Why not a much simpler `workspaceDataviewsConfig.map` and then get the corresponding dataview for each `workspaceDataviewConfig`?
My idea was that a `workspaceDataviewConfig` would always be needed, not optional. So `workspace.dataviewsConfig` should have a `dataviewConfig` for the background and the basemap, event though it only uses the dataviews defaults.